py_mmd_tools/nc_to_mmd.py

```diff
@@ -1155,13 +1155,13 @@ def to_mmd(self, collection=None, checksum_calculation=False, mmd_yaml=None,
 
         mmd_docs = 'https://htmlpreview.github.io/?https://github.com/metno/mmd/blob/master/' \
                    'doc/mmd-specification.html#collection-keywords'
-        default_collections = ['ADC', 'METNCS']
-        if collection is None:
-            logging.warning('Using default values [%s, %s] for the MMD collection field. '
+        default_collection = 'METNCS'
+        if collection is None or collection == "":
```

Comment:
```suggestion
        if collection == "":
```

Reply:
Take another look at line 1136. It says `if collection is *not* None`